repose-aggregator/core/core-lib/src/main/java/org/openrepose/powerfilter/intrafilterLogging/RequestLog.java

```diff
@@ -41,17 +44,25 @@
     String requestBody;
     HashMap<String, String> headers;
 
+    /**
+     * Constructor populates all of the fields necessary for logging.
```

Comment:
My apologies.  No documentation is better than poor documentation.  I'll push an update.